### DISCUSSION TITLE
Don't have access to my own projects

### DISCUSSION THREAD

**Discussion by Harris6144** [AntlerVC/firetable#1215] [Support / Q&A] [answered]
Hello,

Whenever I attempt to create a project, it creates it but then takes me to screen that says I don't have permissions to access it.

Thanks,
Harris.

**Comment by BeeBombshell** [comment]
Hey @Harris6144, does your account has the OWNER / ADMIN permissions? Are you creating a project for the first time?

**Reply by thestonedape** [reply]
same problem and yes my account has owner/ admin permissions

**Comment by BeeBombshell** [comment]
Hey @Harris6144, and @thestonedape... Thank you for pointing this out. We are currently investigating this issue and will get back to you with a solution ASAP. Thank you for your patience. :)

**Reply by BeeBombshell** [reply, marked as answer]
@Harris6144, @thestonedape this should be fixed now... The issue was introduced with the new release so we have reverted to the previous version of the rowy.app. Check and do let me know if this works for you now :)

**Reply by QuirkySwirl** [reply]
It worked for me with a hard browser refresh (shift+refresh)...thanks
